pkgs/development/compilers/ghc/common-hadrian.nix

```diff
@@ -55,7 +55,7 @@
   gmp,
 
   # If enabled, use -fPIC when compiling static libs.
-  enableRelocatedStaticLibs ? stdenv.targetPlatform != stdenv.hostPlatform,
+  enableRelocatedStaticLibs ? stdenv.targetPlatform != stdenv.hostPlatform && !stdenv.targetPlatform.isWindows,
```

Comment:
The formatting is fixed in the last commit, but the change is introduced in the first. Let's clean that up and fix the formatting in the fix commit already.